sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PortEvaluationStage.java

```diff
@@ -18,13 +18,20 @@
     private final String portName;
     private final int port;
 
-    public PortEvaluationStage(Protos.Resource resource, String taskName, String portName, int port) {
+    public PortEvaluationStage(
```

Comment:
Nit: prefer not breaking lines unless the character limit is exceeded? Vertical screen real estate is always the comparatively limited one.